exonum-java-binding/launcher-plugin/README.md

```diff
@@ -0,0 +1,45 @@
+# Java Runtime Plugin for Exonum Launcher
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+- [`exonum-python-client`](https://github.com/exonum/exonum-python-client)
+- [`exonum-launcher`](https://github.com/exonum/exonum-launcher)
+
+```bash
+cd exonum-java-binding
```

Comment:
Probably cd is irrelevant.

Reply:
I just wanted to show current working directory. Probably yes, removed